src/App/Fossa/Compatibility.hs

```diff
@@ -27,9 +26,10 @@ compatibilityMain ::
   [Argument] ->
   IO ()
 compatibilityMain args = withDefaultLogger SevInfo . runExecIO . withCLIv1Binary $ \v1Bin -> do
+  dir <- sendIO getCurrentDir
   cmd <- runStickyLogger SevInfo $ do
     logSticky "[ Waiting for fossa analyze completion ]"
-    exec [reldir|.|] $ v1Command v1Bin args
+    exec dir $ v1Command v1Bin args
```

Comment:
[nit] Should this be target directory?

Reply:
When running syft, there is no target directory.  The positional argument to the command line is a a path or name that syft resolves itself.

Reply:
This is getting removed soon, let's not try to fix it.